api/v1beta2/foundationdbcluster_types.go

```diff
@@ -918,6 +918,10 @@ type FoundationDBClusterAutomationOptions struct {
 
 	// MaintenanceModeOptions contains options for maintenance mode related settings.
 	MaintenanceModeOptions MaintenanceModeOptions `json:"maintenanceModeOptions,omitempty"`
+
+	// IgnoreLogGroupsForUpgrade defines the list of LogGroups that should be ignored during fdb version upgrade.
+	// +kubebuilder:validation:MaxItems=10
+	IgnoreLogGroupsForUpgrade []string `json:"ignoreLogGroupsForUpgrade,omitempty"`
```

Comment:
In a follow up we should define a typed string for the log group to provide some validation patterns for those strings.

Reply:
Could you create a GitHub issue for this?

Reply:
done